perfaware/sim86/shared/Sim8086.cpp

```diff
@@ -73,6 +110,85 @@ static void SimulateInstruction(const instruction& Instruction)
                 } break;
             }
         } break;
+        case Op_add:
+        case Op_sub:
+        case Op_cmp:
+        {
+            const instruction_operand& Dest = Instruction.Operands[0];
+            const instruction_operand& Source = Instruction.Operands[1];
+            u16 Value;
+            u16 Result;
```

Comment:
Ah, I didn't consider invalid inputs because I assumed Muratori would never provide one.
I can see how that's a bad assumption.
The variables are now initialized 
1611dd353481f48f3f0d9b6a2e72b3444b161c2e
and I've added default cases with asserts everywhere
540602ee28dd79c145c86fd2359e2e0b5c6dafd9